src/transformers/training_args.py

```diff
@@ -109,6 +109,8 @@ class TrainingArguments:
             make use of the past hidden states for their predictions. If this argument is set to a positive int, the
             ``Trainer`` will use the corresponding output (usually index 2) as the past state and feed it to the model
             at the next training step under the keyword argument ``mems``.
+        name (:obj:`str`, `optional`, defaults to :obj:`None`):
```

Comment:
Absolutely opinionated nitpick: Is `name` the most descriptive? I wouldn't understand its purpose without reading the description, maybe something like `run_name` or `run_identifier` would be more precise?

Reply:
I like both `run_name` and `run_identifier` better than `name`. Good suggestion!